projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -96,6 +96,20 @@ export const JETPACK_BUNDLES = [
 	PLAN_JETPACK_SECURITY_REALTIME,
 	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
 ];
+export const JETPACK_PLANS_WITH_BACKUP = [
```

Comment:
In the site endpoint response payload, Backup (and other products) are not present in the site features set. Therefore, it seems that the only way to know if a site has a Backup subscription is to check its plan or products slugs directly. That's why I've added this array of plans that include Backup.